### PR TITLE
WIP: Move some more AT's across

### PR DESCRIPTION
**Pull request by markxnelson**
WIP - do not merge yet

### CODE REVIEW

File: tests/e2e/examples/todo_list/todo_list_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package todo_list
```
Comment (mgianatagh) [inline tests/e2e/examples/todo_list/todo_list_suite_test.go:1]: copyright header missing

File: tests/e2e/examples/todo_list/todo_list_test.go [hunk 2]
```diff
@@ -0,0 +1,157 @@
+package todo_list
```
Comment (mgianatagh) [inline tests/e2e/examples/todo_list/todo_list_test.go:1]: copyright header

File: tests/e2e/examples/todo_list/todo_list_test.go [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+import (
+	"fmt"
+	"github.com/onsi/gomega/types"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
```
Comment (mgianatagh) [inline tests/e2e/examples/todo_list/todo_list_test.go:10]: we removed this "." notation from other tests moved over

File: tests/e2e/pkg/environment.go [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+	} else {
+		ingressAddr := fmt.Sprintf("%s:%d", addrHost, addrPort)
+		fmt.Printf("KIND ingress address is %s\n", ingressAddr)
+		return ingressAddr
+	}
+}
+
+func findIstioIngressGatewaySvc(requireLoadBalancer bool) corev1.Service {
+	svcList := ListServices(istioSystemNamespace)
+	//svcList, _ := clientset.Namespace(istioSystemNamespace).ListServices()
```
Comment (mgianatagh) [inline tests/e2e/pkg/environment.go:57]: remove this line

File: tests/e2e/verify-infra/vmi/vmi_test.go [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+				gomega.Eventually(elasticConnected, elasticWaitTimeout, elasticPollingInterval).Should(gomega.BeTrue(), "never connected")
+			},
+			func() {
+				gomega.Eventually(elasticIndicesCreated, elasticWaitTimeout, elasticPollingInterval).Should(gomega.BeTrue(), "indices never created")
+			},
+		)
+	})
+
+	ginkgo.PIt("Elasticsearch filebeat Index should be accessible", func() {
+		//Disabled for KiND. See VZ-1918
```
Comment (mgianatagh) [inline tests/e2e/verify-infra/vmi/vmi_test.go:116]: Remove reference to a Jira?

File: tests/go.mod [hunk 6]
```diff
@@ -4,22 +4,20 @@
...
 	golang.org/x/net v0.0.0-20201021035429-f5854403a974 // indirect
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
-	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	istio.io/api v0.0.0-20200629210345-933b83065c19
 	istio.io/client-go v0.0.0-20200630182733-fd3f873f3f52
-	k8s.io/api v0.18.2
-	k8s.io/apiextensions-apiserver v0.18.2
-	k8s.io/apimachinery v0.18.2
-	k8s.io/client-go v0.18.2
+	k8s.io/api v0.19.0
```
Comment (mgianatagh) [inline tests/go.mod:16]: Are we intentionally updating to v0.19 of k8s.io?

Comment (markxnelson) [reply tests/go.mod:22]: fixed - after a long fight with `go mod`

**Review by gilbode — APPROVED** [review]

File: tests/e2e/verify-infra/vmi/vmi_test.go [hunk 7]
```diff
@@ -0,0 +1,288 @@
...
+	waitTimeout            = 10 * time.Minute
+	pollingInterval        = 30 * time.Second
+	elasticWaitTimeout     = 2 * time.Minute
+	elasticPollingInterval = 5 * time.Second
+)
+
+var _ = ginkgo.BeforeSuite(func() {
+	var err error
+
+	//env = NewVerrazzanoEnvironmentFromConfig(GetTestConfig())
```
Comment (mgianatagh) [inline tests/e2e/verify-infra/vmi/vmi_test.go:69]: this line can be removed

File: tests/go.mod (hunk 6 above)
Comment (mgianatagh) [reply tests/go.mod:22]: I am not sure if you won or lost the fight, which version of k8s.io should be this be.  I presume it should be 0.18.2

Comment (mgianatagh) [reply tests/go.mod:22]: Never mind, I hadn't seen the replace section yet.

**Review by mgianatagh — APPROVED** [review]

**Pull request merged**